capstone/src/main/webapp/script.js

```diff
@@ -12,6 +12,21 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/**
+ * Collect the user's input and call Perspective on it
+ */
+async function submitInput() {
+  const textElement = document.getElementById('textForAnalysis');
+  if (!textElement) {
+    return;
+  }
+  const langElement = document.getElementById('languageForAnalysis');
+  if (!langElement) {
+    return;
+  }
+  const response = await callPerspective(textElement.value, langElement.value);
```

Comment:
Could we start this line at `await...`? Seems like we don't use the `response` constant

Reply:
Yes, that's right - I'll remove the `response`.